docs/framework/wcf/feature-details/security-protocols-version-1-0.md

```diff
@@ -63,7 +63,7 @@ The Web Services Security Protocols provide Web services security mechanisms tha
 |Prefix|Namespace|  
```

Comment:
I think those should remain http as they're.

Reply:
I agree with @Youssef1313 here. Also the change from http to https in the other cells was a mistake as a well.

One option here would be to lock the URL inside backticks so it's treated as code instead of URL, or we could keep as-is knowing these represent namespace values.

Reply:
Makes sense; will change back 👍